### ISSUE TITLE
User Story: Keyboard navigation

### ISSUE THREAD

**Issue by nrsa222** [uklibraries/exploreuk-web-app#44]
As an assisted technology EUK web app user, I want to be able to navigate the entire website using only my keyboard, so that I have the same access to all the content as every other EUK web app user and to comply with ADA and WCAG 2.1 standards.

**Comment by cokernel** [comment]
This feels like two stories, because—I suspect—patrons do not usually care whether we are adhering to regulatory requirements. If we fail to adhere to regulatory requirements, then that may cause a consequence that a patron will care about.

I am thinking of stories along these lines:

* As a patron, I want to be able to navigate the site with my keyboard, so that I have access to all parts of the site. (Meta-story: As a patron, I want to be able to navigate the site with tools I can use, so that I have access to all parts of the site.)

* As a manager of a website, I want users to be able to navigate the site with a keyboard, so that we are in compliance with ADA and WCAG 2.1 standards. (Meta-story: As a manager of a website, I want to adhere to regularity requirements, in particular ADA and WCAG 2.1, so that our site is accessible to as many users as possible, and further so that we minimize the risk of lawsuits.)

I understand that these tests have a shared behavior in keyboard navigability, but they have distinct values.

I am not proposing splitting this *issue* into two issues, because I believe addressing one story will address or at least contribute to addressing the other.
